lib/opencog-chatbot.conf

```diff
@@ -22,19 +22,19 @@ PROMPT                = ""
 
 # These file paths assume that you have done `make install` after
 # compiling; these modules are found in `/usr/local/lib/opencog`.
-MODULES               = opencog/modules/libbuiltinreqs.so,
-                        opencog/modules/libscheme-shell.so,
-                        opencog/modules/libpy-shell.so,
-                        opencog/modules/libattention-types.so,
-                        opencog/modules/libembodiment-types.so,
-                        opencog/modules/libnlp-types.so,
-                        opencog/modules/libspacetime-types.so,
-                        opencog/modules/libLGDictModule.so,
-                        opencog/modules/libPersistModule.so,
-                        opencog/modules/libPythonModule.so,
-                        opencog/modules/libQueryModule.so,
-                        opencog/modules/libRuleEngineModule.so,
-                        opencog/modules/libSuRealModule.so
+MODULES               = opencog/cogserver/modules/libbuiltinreqs.so,
+                        opencog/cogserver/modules/libscheme-shell.so,
+                        opencog/cogserver/modules/libpy-shell.so,
+                        opencog/cogserver/modules/libattention-types.so,
+                        opencog/cogserver/modules/libembodiment-types.so,
+                        opencog/cogserver/modules/libnlp-types.so,
+                        opencog/cogserver/modules/libspacetime-types.so,
+                        opencog/cogserver/modules/libLGDictModule.so,
+                        opencog/cogserver/modules/libPersistModule.so,
+                        opencog/cogserver/modules/libPythonModule.so,
+                        opencog/cogserver/modules/libQueryModule.so,
+                        opencog/cogserver/modules/libRuleEngineModule.so,
+                        opencog/cogserver/modules/libSuRealModule.so
```

Comment:
well, and it did bork the chatbot, and it took me all day to find this, and all the other breakages.  I'm kind of miffed that this pull request was merged, despite the line notes that said "don't merge this".